content/chainguard/chainguard-images/working-with-images/images-directory/index.md

```diff
@@ -109,15 +120,15 @@ Note that Chainguard began generating SBOMs for its Images on November 15, 2023.
 
 ### Vulnerabilities Tab
 
-The last tab is the **Vulnerabilities** tab. This tab contains a list of every CVE one can find within the Image.
+The **Vulnerabilities** tab contains a list of every CVE one can find within the Image.
 
 As with the SBOMs tab, the Vulnerabilities tab has a search box you can use to find and filter specific vulnerabilities within the Image. There is also a drop-down menu to the left allowing you to select different versions of the Image.
 
 Below these is a table listing the vulnerabilities. However, most Chainguard Images won't show any vulnerabilities for the `latest` version. This isn't an error, as we aim to remove vulnerabilities from Images as soon as they arise.
 
-To illustrate how this table appears when vulnerabilities are actually present, you can select different versions in the drop-down until you find one with a vulnerability. This example shows version `16.1-dev` of the `postgres` Image.
+To illustrate how this table appears when vulnerabilities are actually present, you can select different versions in the drop-down until you find one with a vulnerability. This example shows the `latest` version of the `ruby` Image.
 
-![Screenshot of the postgres Image's "Vulnerabilities" tab, with "16.1-dev" selected in the version drop-down menu. The table shows one vulnerability, "CVE-20234-52071", with a "Medium" severity. This CVE is associated with version 8.5.0-r1 of the "libcurl-opennssl" package.](imgs-dir-6.png)
+![Screenshot of the ruby Image's "Vulnerabilities" tab, with "latest" selected in the version drop-down menu. The table shows two vulnerability: "CVE-2024-41946" and "CVE-2024-41123", both with a "Medium" severity. These CVEs are associated with version 3.3.2 of the "rexml" package.](imgs-dir-G.png)
 
 The Vulnerabilities table has four columns.
```

Comment:
5 -- you're missing "last detected"

Reply:
great catch, this is the kind of thing I was worried I'd miss